src/Events/Events.jsx

```diff
@@ -0,0 +1,252 @@
+import React, { Component } from 'react';
+import { jsonp } from '../util';
+import i18n from '../i18n';
+import sanitizeHtml from 'sanitize-html';
+import {
+	Card,
+	CardMedia,
+	CardContent,
+	Typography,
+	CardActions,
+	Button,
+	withStyles,
+	Hidden,
+} from '@material-ui/core';
+import Container from '../Container/Container';
+import _ from 'lodash';
+import ResponsiveImage from '../ResponsiveImage/ResponsiveImage';
+import { translate } from 'react-i18next';
+
+// Meetup API test console: https://secure.meetup.com/meetup_api/console/?path=/:urlname/events
+// page=3 = number of results to return in a page, only need the first 3 results
+const GET_UPCOMING_EVENTS_URL =
+	'https://api.meetup.com/Code-Star-Night/events?photo-host=public&page=3&sig_id=226887185&status=upcoming&fields=featured_photo&sig=d035ab8a8f521cbb4ef14eaff79a55f23c3d25eb';
+
+const GET_PAST_EVENTS_URL =
+	'https://api.meetup.com/Code-Star-Night/events?desc=true&photo-host=public&sig_id=226887185&status=past&fields=featured_photo&sig=a60e663f0904424f80fda3b00bf31f315889231c';
+
+const styles = {
```

Comment:
We can also put this in a `.css` file to keep the component tidy?

Reply:
Agreed, will fix in feature/events-page